src/InputWithLabel.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+
+export default function InputWithLabel ({todoTitle, handleTitleChange, label, isFocused}) {
+    
+    const inputRef = React.useRef();
+
+    React.useEffect(() => {
+        if(isFocused && inputRef.current) {
```

Comment:
You do not need isFocused in this component. You can use only line 9 on useEffect.